keras/backend/theano_backend.py

```diff
@@ -39,6 +39,9 @@ def placeholder(shape=None, ndim=None, dtype=_FLOATX, name=None):
 
     broadcast = (False,) * ndim
     x = T.TensorType(dtype, broadcast)(name)
+    if not hasattr(x.__class__, 'operators_overridden'):
+        from .ops import override_operators
```

Comment:
backend imports ops, and ops imports backends.. maybe thats the reason for the import error when its at top.